ts/components/wallet/TransactionsList.tsx

```diff
@@ -83,10 +81,10 @@ export default class TransactionsList extends React.Component<Props> {
     return (
       <Row>
         <Left>
-          <Text>
-            {isNew && <IconFont name="io-new" style={styles.newIconStyle} />}
+          <View style={styles.dateStyle}>
```

Comment:
sorry, @matgentili I found a better solution.
The previous one was not perfectly center aligned.
Try this one:
```typescript
<View style={{ flexDirection: "row" }}>
  {isNew && (
    <View
      style={{
        flex: 0,
        paddingRight: 4
      }}
    >
      <BadgeComponent />
    </View>
  )}
  <View>
    <Text note={true} style={{ lineHeight: 18 }}>
      {datetime}
    </Text>
  </View>
</View>;
```
Check it out if it works and if yes move the styles to stylesheet

Reply:
Done and I also fixed the datetime font